lib/assert.js

```diff
@@ -807,11 +869,26 @@ function strict(...args) {
   innerOk(strict, args.length, ...args);
 }
 
+const assertInstance = new Assert({ diff: 'simple', strict: false });
+[
+  'ok', 'fail', 'equal', 'notEqual', 'deepEqual', 'notDeepEqual',
+  'deepStrictEqual', 'notDeepStrictEqual', 'strictEqual',
+  'notStrictEqual', 'partialDeepStrictEqual', 'match', 'doesNotMatch',
+  'throws', 'rejects', 'doesNotThrow', 'doesNotReject', 'ifError',
+].forEach((name) => {
+  const bound = assertInstance[name].bind(assertInstance);
+  ObjectDefineProperty(bound, 'name', { __proto__: null, value: name });
+  assert[name] = bound;
+});
+
 assert.strict = ObjectAssign(strict, assert, {
   equal: assert.strictEqual,
   deepEqual: assert.deepStrictEqual,
   notEqual: assert.notStrictEqual,
   notDeepEqual: assert.notDeepStrictEqual,
 });
 
+assert.strict.Assert = Assert;
 assert.strict.strict = assert.strict;
+
+module.exports.Assert = Assert;
```

Comment:
Let's be consistent here and keep calling it `assert`
```suggestion
assert.Assert = Assert;
```